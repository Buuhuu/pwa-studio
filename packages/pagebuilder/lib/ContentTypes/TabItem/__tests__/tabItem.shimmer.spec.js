import React from 'react';

import { createTestInstance } from '@magento/peregrine';

import TabItemShimmer from '../tabItem.shimmer';

jest.mock('@magento/venia-ui/lib/classify');

test('renders an empty TabItemShimmer component', () => {
    const component = createTestInstance(<TabItemShimmer />);

    expect(component.toJSON()).toMatchSnapshot();
});

test('renders a TabItemShimmer component with minHeight', () => {
    const props = {
        minHeight: '400px'
    };
    const component = createTestInstance(<TabItemShimmer {...props} />);

    expect(component.toJSON()).toMatchSnapshot();
});
