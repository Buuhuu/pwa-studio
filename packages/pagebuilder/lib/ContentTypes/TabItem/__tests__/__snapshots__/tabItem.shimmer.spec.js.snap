// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a TabItemShimmer component with minHeight 1`] = `
<div
  aria-busy="true"
  aria-live="polite"
  className="root_rectangle"
  style={
    Object {
      "border": undefined,
      "borderWidth": undefined,
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "minHeight": "400px",
      "paddingBottom": undefined,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": undefined,
    }
  }
/>
`;

exports[`renders an empty TabItemShimmer component 1`] = `
<div
  aria-busy="true"
  aria-live="polite"
  className="root_rectangle"
  style={
    Object {
      "border": undefined,
      "borderWidth": undefined,
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "minHeight": undefined,
      "paddingBottom": undefined,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": undefined,
    }
  }
/>
`;
